.gitmodules

```diff
@@ -46,3 +46,12 @@
 [submodule "vlsi/hammer-cad-plugins"]
 	path = vlsi/hammer-cad-plugins
 	url = https://github.com/ucb-bar/hammer-cad-plugins.git
+[submodule "tools/dsptools"]
```

Comment:
I was under the impression that dsptools was more on the generators side rather than the tools side. Was I mistaken?

Reply:
It's mostly a library, with a few blocks that I'm probably the only user of. I think it belongs closer to chisel than to rocket, but either place would be OK